wkhtmltopdf/utils.py

```diff
@@ -109,7 +109,7 @@ def wkhtmltopdf(pages, output=None, **kwargs):
 
     return check_output(ck_args, **ck_kwargs)
 
-def convert_to_pdf(filename, header_filename=None, footer_filename=None, cmd_options=None):
+def convert_to_pdf(filename_list, header_filename=None, footer_filename=None, cmd_options=None):
```

Comment:
Can we do both, change the name back to 'filename' and add the list coercing as this will have best backwards compatibility.